resources/migrations/001_update_migrations.yaml

```diff
@@ -10137,6 +10137,85 @@ databaseChangeLog:
               - column:
                   name: dashboard_id
 
+  - changeSet:
+      id: v52.2024-12-16T09:19:00
+      author: crisptrutski
+      comment: Metadata about search indexes
+      preConditions:
+        - onFail: MARK_RAN
+        - not:
+            - tableExists:
+                tableName: search_index_metadata
+      changes:
+        - createTable:
+            tableName: search_index_metadata
+            remarks: Each entry corresponds to some queryable index, and contains metadata about it.
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: engine
+                  type: varchar(254)
```

Comment:
nit: 64 seems sufficient?

Reply:
Sure, I guess we're unlikely to get very verbose here. The size of this table is quite small so I wasn't trying to be too frugal.